### PR TITLE
fix DeprecationWarning's in uarray tests

### PR DESCRIPTION
**Pull request by jborbely**
There were a lot of the following warnings issued when running the tests with numpy >= 1.25 installed:

> DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation. (Deprecated NumPy 1.25.)

### CODE REVIEW

**Review by BlairDHall — APPROVED** [review]
This seems straightforward and correct

**Comment by BlairDHall** [comment]
Hi Joe,

I reviewed and approved #25 and #26. Who has the responsibility for
merging? Is that something that I should do too?

Cheers

**Comment by jborbely** [comment]
Either of us can do the merge.

Before you approved the pull requests, the "merge" button was disabled for me (however, I could force the merge if I really wanted to). Once at least 1 person has approved the pull request, the merge button has been enabled.

**Comment by BlairDHall** [comment]
Ok, I see you have done it, but next time I will save you the effort.

Cheers

**Pull request merged**
